node/common/core/lib/message.d.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import Properties = require('./properties');
+
+declare class Message {
+    properties: Properties;
+    messageId: string;
+    to: string;
+    expiryTimeUtc: Date;
+    lockToken: string;
+    correlationId: string;
+    userId: string;
+
+    constructor(data: Message.BufferConvertable);
+
+    getData(): Message.BufferConvertable;
+    getBytes(): Buffer;
+}
+
+declare namespace Message {
+    type BufferConvertable = Buffer | String | any[] | ArrayBuffer;
```

Comment:
Changed to Convertible. Prefer to leave it be, this are the types that can be used in NodeJS Buffer constructor, and so if the that changes in later Node versions we can easily update all occurrences by just changing this line.